src/processing/resources/page.js

```diff
@@ -118,20 +118,13 @@ class PageProcessor extends ResourceProcessorBase {
                 // TODO: Figure out how to emulate the tag behavior.
                 if (domAdapter.getAttr(meta, 'name') === 'referrer')
                     parse5Utils.setAttr(meta, 'content', 'unsafe-url');
-                // NOTE: Remove the existing ‘compatible’ meta tag and add a new one at the beginning of the head.
+                // NOTE: Remove unnecessary existing ‘compatible’ meta tag.
                 if (domAdapter.getAttr(meta, 'http-equiv') === 'X-UA-Compatible')
```

Comment:
I propose to not remove an existing `meta` tag.
In this case we will not change nodes order.